localtests/test.sh

```diff
@@ -134,6 +134,16 @@ test_single() {
   echo_dot
   sleep 1
   #
+
+  local callback_return
+  if [ -f $tests_path/$test_name/before.sh ] ; then
```

Comment:
I added a new way for tests to have before.sh/after.sh to expand on our tests. Each of the new tests depend on this.